cra_buildspec.yaml

```diff
@@ -22,9 +22,9 @@ phases:
     commands:
       - echo saving commit id to file
       - export COMMIT_ID=$(git rev-parse --verify HEAD)
-      - echo $COMMIT_ID > build/commit_id.txt
+      - echo $COMMIT_ID > .next/commit_id.txt
```

Comment:
Absolutely not change this! With this change you are pointing to the folder created when yarn dev command is run.

Revert all changes in this file

Reply:
done